src/groups/power_group.jl

```diff
@@ -0,0 +1,342 @@
+
+const PowerGroup{𝔽,M<:AbstractManifold{𝔽},TPR<:AbstractPowerRepresentation} =
+    GroupManifold{𝔽,<:AbstractPowerManifold{𝔽,M,TPR},ProductOperation}
+
+const PowerGroupNested{𝔽,M<:AbstractManifold{𝔽}} = PowerGroup{𝔽,M,NestedPowerRepresentation}
+
+const PowerGroupNestedReplacing{𝔽,M<:AbstractManifold{𝔽}} =
+    PowerGroup{𝔽,M,NestedReplacingPowerRepresentation}
+
+"""
```

Comment:
I would have added the doctoring to the const – is it better to attach it to the constructor?

Reply:
Sure, I'll add a docstring to this alias.